roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -97,14 +97,22 @@
     kubeadm_config_api_fqdn: "{{ apiserver_loadbalancer_domain_name|default('lb-apiserver.kubernetes.local') }}"
   when: loadbalancer_apiserver is defined
 
+- name: kubeadm | Copy etcd ca file as k8s ca
+  command: "cp -T {{ etcd_cert_dir }}/ca.pem {{ kube_config_dir }}/ssl/etcd/ca.crt"
+  changed_when: false
+
+- name: kubeadm | Copy etcd cakey as k8s cakey
+  command: "cp -T {{ etcd_cert_dir }}/ca-key.pem {{ kube_config_dir }}/ssl/etcd/ca.key"
+  changed_when: false
+
```

Comment:
Can we use the copied certificate? How do you think about it.

Reply:
Initial cri-o ports with kube 1.11 output  error due to cakey file path is invalid.
But configuration seems changed. It is not required now.

Reply:
@okamototk  please delete it.